registry/storage/catalog_test.go

```diff
@@ -60,14 +59,55 @@ func setupFS(t *testing.T) *setupEnv {
 	}
 }
 
+func makeRepo(t *testing.T, ctx context.Context, name string, reg distribution.Namespace) {
+	named, err := reference.ParseNamed(name)
+	if err != nil {
+		return
```

Comment:
Tests shouldn't ignore input like this, if it is bad input the test should just panic